src/components/HelpTopicLink/HelpTopicLink.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { Text, TextVariants } from '@patternfly/react-core';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+
+type HelpTopicLinkProps = {
+  topic: string;
+};
+
+export const HelpTopicLink: React.FC<HelpTopicLinkProps> = ({ topic, children }) => {
+  const {
+    helpTopics: { setActiveTopic },
+  } = useChrome();
+
```

Comment:
You should add the ability to enable the topic on component mount and disable on unmount. This would enable the links to be added as enabled topics for a specific page.

Reply:
Updated to enable topic while HelpTopicLink is mounted. Or should we enable inside the `useQuickstartCloseOnUnmount` hook instead? I tested this while the quickstart is open and it doesn't seem to make a difference.

Reply:
So, the idea is every component brings its own help topics with it (enables on mount and disabled on unmount). So on a page if 3 components enable their own help topics and user clicks on one of them then the sidebar will have all the help topics in dropdown selector. This is a bit different from closing the sidebar when user navigates to a different page.